subsys/bluetooth/host/gatt.c

```diff
@@ -861,7 +987,7 @@ int bt_gatt_service_register(struct bt_gatt_service *svc)
 		return err;
 	}
 
-	sc_indicate(&gatt_sc, svc->attrs[0].handle,
+	sc_indicate(NULL, svc->attrs[0].handle,
```

Comment:
Not sure what is your intend with passing NULL here?

Reply:
Passing conn=NULL means to send an indication to all subscribed devices (or store if those are not connected).